### ISSUE TITLE
ignore:line option has no effect on scripts with type of text/template 

### ISSUE THREAD

**Issue by binarykitchen** [ricardofbarros/linter-js-standard#188]
this is probably a rare case. developing a wordpress plugin here using a html file as a template to inject an inline script which works fine. the only problem is that js-standard in my atom editor is complaining with this warning

"Expected an assignment or function call and instead saw an expression. (no-unused-expressions)"

understand, it's because of the `{{{` part which makes sense. but the problem is that the `// jshint ignore:line` part at end of this line does not seem to have any effect.

```js
<script id='tmpl-nf-field-videomail' type='text/template'>
  <div id='nf-field-{{{ data.id }}}' name='nf-field-{{{ data.id }}}' class='{{{ data.classes }}} nf-element videomail'> // jshint ignore:line
   ...
</script>
```

maybe this exotic use case isn't covered in standardjs yet?

**Comment by sonicdoe** [comment]
Since this ultimately revolves around us mistakenly linting non-JavaScript inline scripts, I’ll close this in favor of #182.
